src/templates/utils-common.js

```diff
@@ -24,9 +24,11 @@ export const matchBrowserLocale = (appLocales, browserLocales) => {
 
   // First pass: match exact locale.
   for (const [index, browserCode] of browserLocales.entries()) {
-    if (appLocales.includes(browserCode)) {
-      matchedLocales.push({ code: browserCode, score: 1 - index / browserLocales.length })
-      break
+    for (const appCode of appLocales) {
+      if (appCode.toLowerCase() === browserCode.toLowerCase()) {
+        matchedLocales.push({ code: appCode, score: 1 - index / browserLocales.length })
+        break
+      }
     }
```

Comment:
Actually we should break out of the outer loop so maybe instead:

```suggestion
    const matchedCode = appLocales.find(appCode => appCode.toLowerCase() === browserCode.toLowerCase())
    if (matchedCode) {
      matchedLocales.push({ code: matchedCode, score: 1 - index / browserLocales.length })
      break
    }
```